core/nginx/conf/nginx.conf

```diff
@@ -127,7 +127,7 @@ http {
       {% endif %}
 
       # If TLS is failing, prevent access to anything except certbot
-      {% if KUBERNETES_INGRESS != 'true' and TLS_ERROR and not (TLS_FLAVOR in [ 'mail-letsencrypt', 'mail' ]) %}
+      {% if KUBERNETES_INGRESS != 'true' and TLS_ERROR and not (TLS_FLAVOR in [ 'mail-letsencrypt', 'mail', 'notls' ]) %}
```

Comment:
I don't understand what this does. It either breaks notls if TLS_ERROR is set or does nothing (as it should never be set when notls is true)

Reply:
scratch that then. i still don't get what notls is for. sorry.